docker/cluster-test/cluster-test.Dockerfile

```diff
@@ -18,7 +18,9 @@ RUN rustup install $(cat rust-toolchain)
 COPY . /libra
 RUN docker/cluster-test/compile.sh
 FROM debian:buster
-RUN apt-get update && apt-get install -y openssh-client curl && curl -L "https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
+RUN apt-get update && apt-get install -y openssh-client curl
```

Comment:
nit - curl is already installed in line 12 above.

Reply:
Looks like thats in a separate build image

Reply:
My bad.